benchmarks/Microsoft.AspNetCore.SignalR.Microbenchmarks/AssemblyInfo.cs

```diff
@@ -0,0 +1 @@
+[assembly: BenchmarkDotNet.Attributes.AspNetCoreBenchmark]
```

Comment:
You don't need to change this...i'm just curious. Would does this work if you put it into the csproj file?
```xml
<ItemGroup>
   <AssemblyAttribute Include="BenchmarkDotNet.Attributes.AspNetCoreBenchmark" />
</ItemGroup>
```
If so, you could add this piece of MSBuild into the Benchmark.Sources package so it's automatically applied to any benchmark project that uses it.

Reply:
I don't want things that implicit. There might be a case where configs need to be overridden and having source file is much easier than trying to find some packages target that brought attribute and figure out how to change it.

Reply:
I could've just dropped assembly info into source package too.